nuxeo/blob.py

```diff
@@ -4,13 +4,12 @@
 import mimetypes
 import os
 import sys
+from io import StringIO
 
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO
+from typing import Any, BinaryIO, Dict, Optional, Text, Union
```

Comment:
Could you make these imports optional? We do not want the user to install `typing` when using the client. It is for dev purpose.